### ISSUE TITLE
Batch Norm layers Assertion Error

### ISSUE THREAD

**Issue by ahndroo** [google-research/morph-net#191]
I'm getting an assertion error  in tpu_util.py when I use batchnorm layers.  I noticed that there are two 'ops' associated with the batchnorm layers:
- 'model/conv1_bn_td/cond/ReadVariableOp_2' 
- 'model/conv1_bn_td/cond/ReadVariableOp_2/Switch'

The error happens when the following line is run on the ''model/conv1_bn_td/cond/ReadVariableOp_2/Switch' op:
`assert len(op.inputs) == 1` 
There are two items in this inputlist.  I can simply comment out this line and everything seems to work, but am I doing anything destructive if I do comment this line out?
